### PR TITLE
FIX: Sequences mismatch

### LINKED ISSUE #148: Mismatched sequences when both single and paired reads available per run ID
When a run ID contains both single AND paired sequences, the forward reads of the paired sequences are not saved correctly. What happens is that the forward reads of the paired sequences are replaced by the single read sequences up until the length of the reverse reads. Here is an illustration, where on the left we see what we would expect and the arrow depicts where q2-fondue goes wrong:
```
Single Sequences: ID1, ..., ID10

Paired Sequences: 
	Forward : ID11, ..., ID15 <--> q2-fondue: Forward: ID1, ..., ID5
	Reverse : ID11, ..., ID15
```
**Steps to reproduce:**
1. Fetch sequences of an accession ID that contains both single and paired sequences, e.g. `ERR3018303`, with `fondue get-sequences`.
2. Extract the content of the fetched single and paired read artifacts.
3. Compare the downloaded sequences to the sequences available on NCBI's SRA.

**Expected behaviour:**
The fetched sequences should match the sequences available on NCBI's SRA.

**Actual behaviour:**
Downloaded forward reads of the paired read sequences do not match the data available on SRA (see illustration above).

### PR DESCRIPTION
**Pull request by adamovanja**
closes #148 

**Testing:**
1. Get a run ID that contains both single and paired reads (e.g. `SRR000001`, `SRR000047`, `SRR000013`).
2. Fetch sequences with fixed q2-fondue `get-sequences`.
3. Compare fetched sequences to sequences uploaded on SRA.

### CODE REVIEW

File: q2_fondue/sequences.py [hunk 1]
```diff
@@ -163,24 +163,32 @@ def _run_fasterq_dump_for_all(
...
     elif filename.endswith('_2.fastq'):
         # paired-end _2
-        acc = re.search(r'(.*)_2\.fastq$', filename).group(1)
+        acc = re.search(r'^.*\/(.*)_2\.fastq$', filename).group(1)
         new_name, is_paired = '%s_00_L001_R2_001.fastq' % acc, True
     elif filename.endswith('.fastq'):
         # single-reads
-        acc = re.search(r'(.*)\.fastq$', filename).group(1)
+        acc = re.search(r'^.*\/(.*)\.fastq$', filename).group(1)
         new_name, is_paired = '%s_00_L001_R1_001.fastq' % acc, False
```
Comment (misialq) [inline q2_fondue/sequences.py:181]: This could simply become something like `_01_L001_R1_001.fastq` to distinguish from `_00_L001...` for paired reads.

**Review by misialq — COMMENTED** [review]
Hey @adamovanja, wow, thanks for looking into that and the quick fix 🙌  I tried with one ID and it all worked - I would like to propose an alternative, simpler solution though. Why can't we just name the reads in the "single" scenario slightly differently, e.g. instead of using `00` for the barcode we could use `01` to distinguish those files - these numbers are fake anyways so it does not matter (see my comment inline). That way we wouldn't/shouldn't need to introduce the complexity of creating additional directories. Also, I think that using `shutil.move` has a potential of actually _copying_ the data in some cases, which is something we should absolutely try to avoid as much as we can - there's enough of data copying in this action already 😅 Let me know in case I missed something though...

File: q2_fondue/sequences.py (hunk 1 above)
Comment (adamovanja) [reply q2_fondue/sequences.py:181]: I agree with you that renaming would be a simpler solution. I have not implemented it this way as I had assumed that the number of the barcode has a particular meaning in the Casava naming format. If this is not the case, differentiating between the two with the barcode sounds like the better solution 😅.

**Review by misialq — APPROVED** [review]
Looks good, thanks @adamovanja! 🥇

**Pull request merged**